class/class3.2/src/Triangle.cpp

```diff
@@ -0,0 +1,9 @@
+#include "Triangle.hpp"
+#include <cmath>
+
+Triangle::Triangle(double a, double b, double c) : a(a), b(b), c(c) {}
+
+double Triangle::area() const {
```

Comment:
validation of inputs related to negative values and triangle inequality are needed